packages/dropdowns/README.md

```diff
@@ -3,6 +3,10 @@
 This package includes components relating to dropdowns in the
 [Garden Design System](https://zendeskgarden.github.io/).
 
+**DEPRECATED:** use [`@zendeskgarden/react-dropdowns.next`][dropdowns.next] instead.
+
+[dropdowns.next]: https://github.com/zendeskgarden/react-components/tree/main/packages/dropdowns.next
```

Comment:
I'm tentative about adding this one, since our current intent (pending plan details) is to move `dropdowns.next` to `dropdowns` in the next major. As a result, we don't plan to run `npm deprecate` the `@zendeskgarden/react-dropdowns` package.

Reply:
That's a good point, and I couldn't totally remember if we were doing it that way. I'll remove it from the package README :)

Reply:
Updated: 47c6b0dfc